proposals/0011-resource-element-type-validation.md

```diff
@@ -28,7 +28,7 @@ and valid element types for `*StructuredBuffer` resources and `*ByteAddressBuffe
 
 Element types for typed buffer resources:
 * Are not intangible (e.g., isn't a resource type)
-* Must be vectors or scalars of arithmetic types, not bools nor enums nor arrays
+* Must be vectors or scalars of arithmetic types, not bools nor enums nor arrays nor structs
 * Should be a scalar or homogenous vector of a floating-point or integer type, with a maximum of 4 components after translating 64-bit components into pairs of uint32_t components
 Element types for raw buffer resources:
 * Are not intangible (e.g., isn't a resource type)

```

Comment:
This "Are not intangible" bullet seems to be repeated twice. Mind cleaning that up while you're here?

Reply:
It's intended to be there twice, I am trying to express that both typed buffer and raw buffer resource element types should not be intangible.

Reply:
Ah. I think there's a formatting issue here:

<img width="691" alt="image" src="https://github.com/user-attachments/assets/574a7306-2ab1-4aef-831f-a301b40d5b26">

You probably need a new line between the "typed buffer" bullet points and the "raw buffer" text.

Reply:
You're right, I'll edit that!